templates/ta-assignments.html

```diff
@@ -176,7 +182,13 @@ <h5 class="table-header">Section</h5>
                                     <form class="input-group" name="assignment-ta" action="" method="post">
                                         {% csrf_token %}
                                         <span class="tlabel">Email:</span>
-                                        <input type="email" name="section_ta_email" class="form-control" placeholder="TA Email" required>
+                                        <select type="email" name="section_ta_email" class="form-control" placeholder="TA Email" required>
+                                            {% for user in TAs %}
+                                                {% if user.ROLE == 'TA' %}
+                                                    <option value=" {{ user.EMAIL }}">{{ user.EMAIL }}</option>
```

Comment:
Oh, that was actually for course email, but it's the same issue causing it for adding TAs.